zerver/migrations/0055_realm_email_changes_disabled.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-02-27 14:34
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
I think you'll need to renumber this, since we've merged a first emoji since then.  See http://zulip.readthedocs.io/en/latest/migration-renumbering.html